### PR TITLE
change ocis basepath from home to

### PR DESCRIPTION
**Pull request by dragonchaser**
This PR changes the basepath for the docs helper from `~` (HOME) to `/etc`

refs #7972

### CODE REVIEW

File: docs/helpers/configenvextractor.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func generateIntermediateCode(templatePath string, intermediateCodePath string,
 
 func runIntermediateCode(intermediateCodePath string) {
 	fmt.Println("Running intermediate go code for " + intermediateCodePath)
-	defaultPath := "~/.ocis"
```
Comment (rhafer) [inline docs/helpers/configenvextractor.go:66]: Won't this result in /etc/ocis/config for the default config path and /etc/ocis/ for the datapath? That sounds wrong.

**Review by rhafer — CHANGES_REQUESTED** [review]

**Review by kobergj — APPROVED** [review]
Is this a breaking change? Or it was just broken before anyways?

**Comment by dragonchaser** [comment]
> Is this a breaking change? Or it was just broken before anyways?

It will just change the output in the docs.

File: docs/helpers/configenvextractor.go (hunk 1 above)
Comment (dragonchaser) [reply docs/helpers/configenvextractor.go:66]: Uhm you are right....
will fix

Comment (kobergj) [reply docs/helpers/configenvextractor.go:66]: Config path is adjusted later. This is just the base path
```
os.Setenv("OCIS_CONFIG_DIR", path.Join(defaultPath, "config"))
```

Comment (dragonchaser) [reply docs/helpers/configenvextractor.go:66]: The defaultPath would be `~/.ocis` resulting in `~/.ocis/config` for the config path.
On a container it the config path would be plain `/etc/ocis` and the respective datapath `/var/lib/ocis`.

Comment (dragonchaser) [reply docs/helpers/configenvextractor.go:66]: @rhafer should be fixed now

Comment (rhafer) [reply docs/helpers/configenvextractor.go:66]: BTW, for our release build we set the paths via build flags. Maybe you can utilize that here as well?

```
-X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseDataPathType=path" -X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseDataPathValue=/var/lib/ocis"
-X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseConfigPathType=path" -X "$(OCIS_REPO)/ocis-pkg/config/defaults.BaseConfigPathValue=/etc/ocis"
```

**Review by rhafer — APPROVED** [review]

**Pull request merged**